.github/ISSUE_TEMPLATE/1_Bug_report.md

```diff
@@ -10,7 +10,7 @@ about: Report errors and problems
 
 | Subject        | Details                                                         |
 | :------------- | :---------------------------------------------------------------|
-| Rector version | e.g. v0.9.5 (invoke `vendor/bin/rector --version`)              |
+| Rector version | e.g. v0.11.5 (invoke `vendor/bin/rector --version`)              |
 | Installed as   | composer dependency / prefixed Rector                           |
```

Comment:
this line can be dropped, because there is only a single install method as of 0.11, right?

Reply:
Yes, it's from the old times :laughing: